docs/OTP2-MigrationGuide.md

```diff
@@ -67,98 +83,119 @@ These parameters are no longer supported:
  - `timeouts` OTP1 searches the graph many times, while OTP2 do one search finding multiple results
-              in one search. So, there is no need for this parameter.
- - `boardTimes` is replaced by `request` parameter `boardSlack` and `boardSlackForMode`.
- - `alightTimes` is replaced by `request` parameter `alightSlack` and `alightSlackForMode`.
+              in one search. So, there is no need for this parameter. Since v2.0
+ - `boardTimes` is replaced by `request` parameter `boardSlack` and `boardSlackForMode`. Since v2.0
+ - `alightTimes` is replaced by `request` parameter `alightSlack` and `alightSlackForMode`. Since v2.0
  
    
 ## REST API
-  
+
 ### Trip Planning 
 
 Support for XML as a request/response format is removed. The only supported format is JSON.
+Some of these parameters may only be available as `defaultRequest` configuration parameters.
+
 
 #### Query parameter changes
 
-A lot of the query parameters in the REST API are ignored/deprecated, see the [RoutingRequest](https://github.com/opentripplanner/OpenTripPlanner/blob/v2.0.0/src/main/java/org/opentripplanner/routing/api/request/RoutingRequest.java) 
- and the [RoutingResource](https://github.com/opentripplanner/OpenTripPlanner/blob/v2.0.0/src/main/java/org/opentripplanner/api/common/RoutingResource.java)
- class for the documentation on what is now supported in 2.0. A few features did not make it into 
- the 2.0, but is sheduled for the 2.1 release.
- See [GitHub issues 2.1](https://github.com/opentripplanner/OpenTripPlanner/issues?q=is%3Aopen+is%3Aissue+milestone%3A2.1).  
+A lot of the query parameters in the REST API are ignored/deprecated, see the [RoutingRequest](https://github.com/opentripplanner/OpenTripPlanner/blob/v2.1.0/src/main/java/org/opentripplanner/routing/api/request/RoutingRequest.java) 
+ and the [RoutingResource](https://github.com/opentripplanner/OpenTripPlanner/blob/v2.1.0/src/main/java/org/opentripplanner/api/common/RoutingResource.java)
+ class for the documentation on what is now supported in OTP2.  
  
  
-The following parameters are missing in OTP2 but will be added:
- 
-  - `startingTransitTripId` The ability to plan a trip from on board a vehicle should be implemented 
-    by Q1 2021.
-  - `intermediatePlaces` - ability to specify intermediate destinations along the route. It is not 
-    certain when this will be implemented. 
-  - `nonpreferredTransferCost`, `(un)preferredRoutes`, `(un)preferredAgencies` - these help diversify
-    or customize the trips and operators visible in results. Due to the new transit routing
-    algorithm, Entur plans to   completely rewrite these features, accounting for market-neutrality
-    requirements and showing relevant trips and operators in local vs. intercity trips.
+#### The following parameters are missing in OTP2 but will be added
+
+- `startingTransitTripId` The ability to plan a trip from on board a vehicle should be implemented 
+  by Q1 2021.
```

Comment:
This says it will be implemented by Q1 2021 (one year ago) but the field in RoutingRequest is marked `@Deprecated`.